t/geturl_connectivity.pl.in

```diff
@@ -55,35 +58,52 @@ sub run_httpd {
-    { ipv6_opt => 0, client_ipv => '6', server_ipv => '6', ssl => 1 },
-    { ipv6_opt => 1, client_ipv => '',  server_ipv => '4', ssl => 0 },
-    { ipv6_opt => 1, client_ipv => '',  server_ipv => '4', ssl => 1 },
-    { ipv6_opt => 1, client_ipv => '',  server_ipv => '6', ssl => 0 },
-    { ipv6_opt => 1, client_ipv => '',  server_ipv => '6', ssl => 1 },
-    { ipv6_opt => 1, client_ipv => '4', server_ipv => '4', ssl => 0 },
-    { ipv6_opt => 1, client_ipv => '4', server_ipv => '4', ssl => 1 },
-    { ipv6_opt => 1, client_ipv => '6', server_ipv => '6', ssl => 0 },
-    { ipv6_opt => 1, client_ipv => '6', server_ipv => '6', ssl => 1 },
+    { ipv6_opt => 0, client_ipv => '',  server_ipv => '4', ssl => 0, curl => 0 },
+    { ipv6_opt => 0, client_ipv => '',  server_ipv => '4', ssl => 1, curl => 0 },
+    { ipv6_opt => 0, client_ipv => '4', server_ipv => '4', ssl => 0, curl => 0 },
+    { ipv6_opt => 0, client_ipv => '4', server_ipv => '4', ssl => 1, curl => 0 },
+    { ipv6_opt => 0, client_ipv => '6', server_ipv => '6', ssl => 0, curl => 0 },
+    { ipv6_opt => 0, client_ipv => '6', server_ipv => '6', ssl => 1, curl => 0 },
+    { ipv6_opt => 1, client_ipv => '',  server_ipv => '4', ssl => 0, curl => 0 },
+    { ipv6_opt => 1, client_ipv => '',  server_ipv => '4', ssl => 1, curl => 0 },
+    { ipv6_opt => 1, client_ipv => '',  server_ipv => '6', ssl => 0, curl => 0 },
+    { ipv6_opt => 1, client_ipv => '',  server_ipv => '6', ssl => 1, curl => 0 },
+    { ipv6_opt => 1, client_ipv => '4', server_ipv => '4', ssl => 0, curl => 0 },
+    { ipv6_opt => 1, client_ipv => '4', server_ipv => '4', ssl => 1, curl => 0 },
+    { ipv6_opt => 1, client_ipv => '6', server_ipv => '6', ssl => 0, curl => 0 },
+    { ipv6_opt => 1, client_ipv => '6', server_ipv => '6', ssl => 1, curl => 0 },
+    { ipv6_opt => 0, client_ipv => '',  server_ipv => '4', ssl => 0, curl => 1 },
+    { ipv6_opt => 0, client_ipv => '',  server_ipv => '4', ssl => 1, curl => 1 },
+    { ipv6_opt => 0, client_ipv => '4', server_ipv => '4', ssl => 0, curl => 1 },
+    { ipv6_opt => 0, client_ipv => '4', server_ipv => '4', ssl => 1, curl => 1 },
+    { ipv6_opt => 0, client_ipv => '6', server_ipv => '6', ssl => 0, curl => 1 },
+    { ipv6_opt => 0, client_ipv => '6', server_ipv => '6', ssl => 1, curl => 1 },
+    { ipv6_opt => 1, client_ipv => '',  server_ipv => '4', ssl => 0, curl => 1 },
+    { ipv6_opt => 1, client_ipv => '',  server_ipv => '4', ssl => 1, curl => 1 },
+    { ipv6_opt => 1, client_ipv => '',  server_ipv => '6', ssl => 0, curl => 1 },
+    { ipv6_opt => 1, client_ipv => '',  server_ipv => '6', ssl => 1, curl => 1 },
+    { ipv6_opt => 1, client_ipv => '4', server_ipv => '4', ssl => 0, curl => 1 },
+    { ipv6_opt => 1, client_ipv => '4', server_ipv => '4', ssl => 1, curl => 1 },
+    { ipv6_opt => 1, client_ipv => '6', server_ipv => '6', ssl => 0, curl => 1 },
+    { ipv6_opt => 1, client_ipv => '6', server_ipv => '6', ssl => 1, curl => 1 },
```

Comment:
`fetch_via_curl` doesn't care about `$globals{'ipv6'}`, so omit the `ipv6_opt => 1` cases.

Reply:
Yes, true.  And neither should `fetch_via_socket_io` now that ip version is passed in as a parameter.  But that change should wait until we progress further with IPv6 support -- need my two PRs merged before that can commence.

Reply:
done